### ISSUE TITLE
Scrolling in a short email may trigger left/right swipe

### ISSUE THREAD

**Issue by PhilC813** [thunderbird/thunderbird-android#8376]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

14.26.6

### Where did you get the app from?

Google Play

### Android version

Android 14 / REDMAGICOS9.0.20_NX769J_GB

### Device model

REDMAGIC 9 Pro

### Steps to reproduce

1. Open an email that is short enough to fit in whole on the screen
2. Scroll up/down quickly

### Expected behavior

Nothing happens

### Actual behavior

If the scrolling action wasn't perfectly vertical, it will mistake the gesture for a left/right swipe and switch to the previous/next email.

If improving the gesture detection isn't an option, an easy fix would be to add an option to the settings to disable the left/right gesture within emails.

**Comment by idl0r** [comment]
Please also see #8933 
It's not just with short mails.

**Comment by kewisch** [comment]
I think we should rather decrease the sensitivity than add more options. It seems to be less of an issue on my test device. Maybe if we decrease the area where swiping is effective more towards the border, then scrolling down can occur naturally, and swiping happens closer to the border.

More creative ideas on making it harder to accidentally swipe without changing the touch area would also be welcome.

**Comment by idl0r** [comment]
Yeah, that might be enough already. Not sure how difficult it is. I thought a option/toggle might be really easy.

**Comment by Gehmasse** [comment]
I encounter the same issue and think it would be good for users to have an option to entirely disable this swipe functionality. For me, it's more confusing than helpful because I cannot see which mail comes next without marking it as seen. I prefer opening each mail separately from the mail list. Maybe this workflow is similar for other users.

**Comment by ayus-aryal** [comment]
@kewisch I would like to work on this issue. May I get assigned with the issue?

**Comment by kewisch** [comment]
@ayus-aryal yes, feel free to give it a shot. I think what you could investigate is if there is a way to reduce swipe sensitivity without reducing the touch area. I read something about fling velocity, maybe this applies here as well. I asked design folks and there were some concerns about reducing the touch area as it could inhibit accessibility. Looking at some other clients, swipe actions don’t happen if vertical scroll is currently in motion, and the tap detection is very good. 

Could you check if there is a way to make it less likely for a swipe to occur as noted?

**Comment by joohnq** [comment]
I opened a PR for this feature. Please review it and let me know if any changes are needed.

**Comment by idl0r** [comment]
So to be honest, I still have those issues. Thanks for the effort so far! It might be better but at least to me not enough. A slider or integer value option would be cool or really just a toggle to turn it off entirely.

**Comment by idl0r** [comment]
It is still really frustrating to me :/
An option would be really appreciated .
Can someone please reopen? @PhilC813 by chance?
Esp. Now/still in longer mails, very annoying.

**Comment by PhilC813** [comment]
I'm not irritated as often thanks to the fix, but I do agree that there should be an option, just like the existing ones for left/right swipe in the messages list.

Not a perfect comparison, but the UI of RSS readers is similar to email clients, and the one I use, Capy Reader, has such an option in its settings; I don't see why the same can't be implemented here.

| Capy Reader > Settings > Gestures |
|--------|
| <img width="1440" height="2618" alt="Image" src="https://github.com/user-attachments/assets/bb5f511d-abe0-4a5f-9686-197b64f43211" /> |

It says I don't have permission to reopen the issue though.
@wmontwe what do you think?

**Comment by idl0r** [comment]
Alternatively it must be much harder to swipe. Or some kind of magnetic effect, something to make sure one doesn't accidentally swipe just by scrolling through mails.

**Comment by shamefulCake1** [comment]
Why is this closed? Swiping right/left is super annoying and is not helpful.
